styleguide/Wrapper.js

```diff
@@ -16,7 +17,11 @@ export default class ThemeWrapper extends Component {
       <Fragment>
         <AppGlobalStyles />
         <StyleguideGlobalStyles />
-        <ThemeProvider theme={theme}>{this.props.children}</ThemeProvider>
+        <ThemeProvider theme={theme}>
+          <StripeProvider token={process.env.STRIPE_KEY} loadOnMount>
```

Comment:
How do we ensure that this env variable is available? The `webpackConfig` in `styleguide.config.js`?

Reply:
This code doesn't ensure that the variable is available. If it's not, then the stripe input won't load and we'll get a warning in console saying that the stripe key is missing.

Reply:
What I can do is to check what has been loaded from `.env` in `styleguide.config.js ` and set defaults if necessary